SU-HSDO/suhumsci
# SHS-5171: adjusting titles to include date info

# READY FOR REVIEW

## Summary
This PR adds date information to event titles so events with the same name will have differentiating information within the title's link for accessibility. The date is visually hidden. 

## Urgency
medium

## Steps to Test
1. Set up the Statistics site locally
2. Visit the homepage
3. Note that the date component has multiple events with the same name, but now have hidden text showing the date. 

I was hoping for a cleaner solution, but the link is built in the view. Let me know if you think I should do this by rewriting the title to include the date from in the View itself. 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
